examples/security/v_dra/test.py

```diff
@@ -0,0 +1,114 @@
+import torch
+from data_reconstruction_attack import *
+
+import argparse
+import os
+import torch.backends.cudnn as cudnn
+import torchvision.utils as vutils
+
+from model_resnet import ResNet18
+from model_stylegan2 import Generator
+
+
+def main(args):
+    torch.manual_seed(args.random_seed)
+
+    use_cuda = torch.cuda.is_available()
+    device = torch.device(args.gpu_id if use_cuda else 'cpu')
+
+    cuda_kwargs = {}
+    if use_cuda:
+        torch.cuda.set_device(args.gpu_id)
+        cudnn.benchmark = True
+        cuda_kwargs = {'num_workers': args.workers, 'pin_memory': True}
+
+    # load test data
+    if not os.path.exists(args.data_path):
+        raise Exception("Test Dataset does not exist!")
+    test_dataset = torch.load(args.data_path)
+    test_loader = torch.utils.data.DataLoader(
+        test_dataset, batch_size=1, shuffle=False, **cuda_kwargs
+    )
+
+    print("Test dataset length:", len(test_dataset))
+
+    # load target model
+    target_model = ResNet18()
+    if not os.path.exists(args.trg_model_path):
+        raise Exception("Target Model does not exist!")
+    target_model = torch.load(args.trg_model_path)
```

Comment:
If model initialized, why not use load_state_dict?
If use torch.load, model needn't to be initialized.

Reply:
We have revised this to load_state_dict.